docs/configuration/service/suricata.rst

```diff
@@ -0,0 +1,105 @@
+.. _suricata:
+
+########
+suricata
+########
+
+:abbr:`suricata` Suricata and VyOS are powerful tools for ensuring network security and traffic management. 
+Suricata is an open-source intrusion detection and prevention system (IDS/IPS) that analyzes network packets in real-time. 
+
+
+Suricata Features
+=================
+
+Intrusion Detection (IDS): Analyzes network traffic and detects suspicious activities, attacks, and malicious traffic.
```

Comment:
this points looks odd in the rendered page
look [here](https://vyos--1509.org.readthedocs.build/en/1509/configuration/service/suricata.html).

maybe bullet point will fit better?

Reply:
I just removed the indents between the lines, it made it better.